packages/react-atlas-core/src/Form/Form.js

```diff
@@ -158,30 +158,25 @@ class Form extends React.PureComponent {
   };
 
   submitHandler = e => {
-    /* Prevent form submission if action prop is not set. */
-    if (typeof this.props.action === "undefined") {
+    if(typeof this.props.action === 'undefined') {
```

Comment:
Is this from lint? or typo? line didn't change other than whitespace.

Reply:
Ohh I retyped it, i think it's missing a space now